src/foam/nanos/controller/AppStyles.js

```diff
@@ -14,99 +14,99 @@ foam.CLASS({
   css: `
     body {
       font-family: /*%FONT1%*/ Roboto, 'Helvetica Neue', Helvetica, Arial, sans-serif;
-      font-size: 14px;
-      letter-spacing: 0.2px;
+      font-size: 1rem;
+      letter-spacing: 0.014em;
       color: /*%BLACK%*/ #1E1F21;
       background: /*%GREY5%*/ #f5f7fa;
       margin: 0;
     }
 
-    /* TYPOGRAPHY */
+    /* TYPOGRAPHY */ 
     .h100{
       font-style: normal;
       font-weight: 700;
-      font-size: 35px;
-      line-height: 40px;
+      font-size: 2.5rem;
+      line-height: 1.14;
       margin: 0;
     }
     .h200{
       font-style: normal;
       font-weight: 600;
-      font-size: 29px;
-      line-height: 32px;
+      font-size: 2.07rem;
+      line-height: 1.1;
       margin: 0;
     }
     .h300{
       font-style: normal;
       font-weight: 600;
-      font-size: 24px;
-      line-height: 28px;
+      font-size: 1.71rem;
+      line-height: 1.17;
       margin: 0;
     }
     .h400{
       font-style: normal;
       font-weight: 600;
-      font-size: 20px;
-      line-height: 24px;
+      font-size: 1.43rem;
+      line-height: 1.2;
       margin: 0;
     }
     .h500{
       font-style: normal;
       font-weight: 600;
-      font-size: 16px;
-      line-height: 20px;
+      font-size: 1.14rem;
+      line-height: 1.25;
       margin: 0;
     }
     .h600{
       font-style: normal;
       font-weight: 600;
-      font-size: 14px;
-      line-height: 18px;
+      font-size: 1rem;
+      line-height: 1.29;
       margin: 0;
     }
     .p{
       font-style: normal;
       font-weight: normal;
-      font-size: 14px;
-      line-height: 24px;
+      font-size: 1rem;
```

Comment:
Do we specify 1rem to be 14px somewhere?  Browser default is 16px so this would be incorrect sizing

Reply:
I think we specify it on this line haha
https://github.com/kgrgreer/foam3/pull/666/files#diff-db3347d781c601921aa474ac02646692d58b72247d26f0a29ebbb7acecf65640R17
will change that

Reply:
rem will be specified for the html tag not the body

Reply:
ah ok, then not that i can find, i can add it here though